tests/gui/measure/_defaults/test_aliases.py

```diff
@@ -0,0 +1,92 @@
+"""Tests for _aliases.ini of viperleed.gui.measure._defaults."""
+
+__authors__ = (
+    'Michele Riva (@michele-riva)',
+    'Florian Dörr (@FlorianDoerr)',
+    )
+__copyright__ = 'Copyright (c) 2019-2025 ViPErLEED developers'
+__created__ = '2025-10-02'
+__license__ = 'GPLv3+'
+
+import ast
+from configparser import ConfigParser
+
+from pytest_cases import fixture
+
+from viperleed.gui.measure.classes.settings import get_default_aliases_path
+
+
+@fixture
+def alias_config():
```

Comment:
If you run pylint on this file, it will tell you that your test functions "mask" the "global" alias_config name. A way to prevent that is the following:
```suggestion
@fixture(name='alias_config')
def fixture_alias_config():
```